src/components/AnalysisModal.js

```diff
@@ -104,8 +104,8 @@ export const AnalysisModal = Utils.withDisplayName('AnalysisModal')(
     const getEnvironmentView = () => Utils.switchCase(currentTool,
       [tools.Jupyter.label, renderComputeModal],
       [tools.RStudio.label, renderComputeModal],
-      [tools.galaxy.label, renderAppModal(GalaxyModalBase, tools.galaxy.label)],
-      [tools.cromwell.label, renderAppModal(CromwellModalBase, tools.cromwell.label)]
+      [tools.galaxy.label, () => renderAppModal(GalaxyModalBase, tools.galaxy.label)],
```

Comment:
FYI @kyuksel @jdcanas this fixes a current bug introduced by my PR (a late refactoring in the review process that I didn't adequately test), where the "Create" button does not work for Galaxy and Cromwell. Creating an instance through the side cloud environment modal DOES work.

Just want to make sure you guys don't waste time filing a bug/investigating this if you see it. Hopefully this change will be out early next week, and I could also submit it as a separate PR if this one gets further delayed and folks are hitting this.